examples/api/lib/material/stepper/step_style.0.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
You could demo the copyWith and merge methods with this example. For example you could create a StepStyle that defined a set of basic visual properties and then use copyWith to make a few adjustments for each step.